### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**
Please check my readme

### CODE REVIEW

File: shell-and-git/github-and-markdown.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
-- ![Link] (url) Image
\ No newline at end of file
+- ![Link] (url) Image
+
+# Git Commands:
+
+- git status: Show status of branch
+- git add: Add file or folder to staged
+- git commit -m \<message>: Commitment to git
+- git branch: List your branches
```
Comment (Sukhpreet92) [inline shell-and-git/github-and-markdown.md:12]: Add more details.

Comment (Roland-Hufnagel) [reply shell-and-git/github-and-markdown.md:12]: okay

**Review by AHen-C0de — APPROVED** [review]
Looks good!

File: shell-and-git/github-and-markdown.md (hunk 1 above)
Comment (Sukhpreet92) [reply shell-and-git/github-and-markdown.md:12]: 🆗

**Pull request merged**